config/default.yml

```diff
@@ -198,6 +198,12 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', '^<<<<<<<\s']
 
+  Pep257:
```

Comment:
Sorry, I specifically meant PEP257 (I'm familiar with PEP in general).

However, having dug into the spec this makes sense as a default.